src/components/ArticlesCollection/ArticlesCollection.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Thumbnail } from "../Thumbnail/Thumbnail";
+import homer from "../../static/svg/homer.svg";
+import { Button } from "../Button";
+
+export const AllArticles = () => {
```

Comment:
I see the file was renamed 👍the const itself should be the same `ArticlesCollection`